index.js

```diff
@@ -1,16 +1,18 @@
 
 var mcss = require('mcss'),
-  loaderUtils = require('loader-utils')
+  loaderUtils = require('loader-utils');
 
 module.exports = function(content) {
 	this.cacheable && this.cacheable();
 
 	var callback = this.async(),
-    options = loaderUtils.parseQuery(this.query)
-	
-	mcss(options).translate(content).done(function(text) {
+        options = loaderUtils.parseQuery(this.query);
+
+	options.filename = this.resource;//fix the path bug: @import string contain char like '../../'
```

Comment:
like the less-loader，you should set filename to options for mcss parse the keyword @import or @abstract